continuous_integration/test_script.sh

```diff
@@ -29,8 +31,8 @@ else
 fi
 
 # Is directory still empty ?
-ls
+ls -ltra
```

Comment:
Apparently this has something to do with Debian packaging policy, not an expert on this. Maybe someone who know more than me should update the comment above the ls.